tools/snp.sh

```diff
@@ -973,8 +1138,11 @@ main() {
   done
   
   # Set SETUP_WORKING_DIR for non-upm
+    # COMMENT  
+    # I haven't tested 'sev-snp-devel'(no-upm) branch 
+    # as I was not sure if I can test this branch as I already installed snp kernel package from snp-latest branch
   if ! $UPM; then
-    SETUP_WORKING_DIR="${SETUP_WORKING_DIR}/non-upm"
+    SETUP_WORKING_DIR="${SETUP_WORKING_DIR}/sev-snp-devel"
```

Comment:
Small Question

Is this AMDSEV/sev-snp-devel branch actively used now? 
Asking this as I need to make changes in the sev-snp-devel branch for RedHat and Fedora environment.
Till now I focused on snp-latest branch